### DISCUSSION TITLE
zh/soft/web

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#960] [Announcements]
# zh/soft/web

企业门户网站 概念 业界认为企业门户就是一个联接企业内部和外部的网站，它可以为企业提供一个单一的访问企业各种信息资源的入口，企业的员工、客户、合作伙伴和供应商等等都可以通过这个门户获得个性化的信息和服务。企业门户可以无缝地集成企业的内容、商务和社区：首先，通过企业门户，企业能够动态地发布存储在企业内部和外部的各种信息；其次，企业门户可以完成网上的交易；...

https://topmap.net/zh/soft/web.html

**Comment by hbtykj** [comment]
测试
